internal/errors/http.go

```diff
@@ -29,3 +29,8 @@ func NewHTTPErrorF(code int, format string, a ...any) error {
 func NilArgError(name string) error {
 	return NewHTTPErrorF(http.StatusInternalServerError, "'%s' cannot be nil", name)
 }
+
+// EmptyArgError creates a new HTTPError instance with "'name' cannot be empty"
+func EmptyArgError(name string) error {
+	return NewHTTPErrorF(http.StatusBadRequest, "'%s' cannot be empty", name)
+}
```

Comment:
I would suggest a unit test function, for instance the below one:

```go
func TestEmptyArgError(t *testing.T) {
	err := EmptyArgError("param")
	he := httpErrorFromErr(t, err)
	assert.Equal(t, he.Code, int(http.StatusBadRequest))
	require.IsType(t, "string", he.Message)
	msgString, ok := he.Message.(string)
	require.True(t, ok)
	assert.Equal(t, "'param' cannot be empty", msgString)
	assert.NoError(t, he.Internal)
}
```

Reply:
Added